File: problemas/iniciante/1015/1015.c
#include <stdio.h>
#include <math.h>

int main()
{
    double x1, y1, x2, y2, distancia;

    scanf("%lf %lf\n%lf %lf", &x1, &y1, &x2, &y2);

    distancia = sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1));

    printf("%.4lf\n", distancia);

    return 0;
}

File: problemas/iniciante/1015/1015.ml
Scanf.scanf "%f %f %f %f" (fun x1 y1 x2 y2 ->
    Printf.printf "%.4f\n" (sqrt(((x2 -. x1) *. (x2 -. x1)) +. ((y2 -. y1) *. (y2 -. y1)))))
